examples/port-2-alibaba2-connection/README.md

```diff
@@ -0,0 +1,69 @@
+# Fabric Port to Fabric Alibaba Profile Connection
+
+This is a step-by-step guide for creating a connection using the Equinix Terraform Provider and the Alibaba Terraform Provider.
+By following these steps, you will be able to:
+
+1. Setup a connection through Equinix Fabric.
+2. Configure and accept the connection in the Alibaba Portal.
+3. Manage resources effectively using Terraform.
+4. Perform cleanup operations efficiently
+
+### Step by Step Instructions for Fabric Port to Fabric Alibaba Profile Connection
+
+#### 1. Create Connection from Equinix Terraform Provider
```

Comment:
This header needs to be placed below the standard README.md content we usually have in the header.